src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -121,33 +122,31 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
         return (this._inputs[deviceType] !== undefined);
     }
 
-    /**
-     * Dispose of all the eventlisteners
-     */
-    public dispose(): void {
-        // Observables
-        this.onDeviceConnected = () => { };
-        this.onDeviceDisconnected = () => { };
-        this.onInputChanged = () => { };
-
-        if (this._elementToAttachTo) {
-            this._removeEvents();
-
-            // Gamepad Events
-            window.removeEventListener("gamepadconnected", this._gamepadConnectedEvent);
-            window.removeEventListener("gamepaddisconnected", this._gamepadDisconnectedEvent);
-        }
-    }
-
     /**
      * Configures events to work with an engine's active element
      */
-    private _configureEvents(): void {
+     public configureEvents(): void {

```

Comment:
These names are asymmetrical. Either configure/unconfigure or match the naming from DeviceSourceManager?

Reply:
Changing to configure/unconfigure